### DISCUSSION TITLE
字距和格式

### DISCUSSION THREAD

**Discussion by Ellyc4096** [rime/home#997] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43885814/149403130-c840a6cf-9590-465d-a6df-164f7ce87994.png)
![image](https://user-images.githubusercontent.com/43885814/149403185-bace6da4-4419-474d-a1fc-4cf85385d437.png)
打英文就會有超大字距
空格也很大
為甚麼會這樣 T.T

**Comment by LEOYoon-Tsaw** [comment, marked as answer]
用全角了吧

**Reply by Ellyc4096** [reply]
靠原來這麼簡單
謝了
